### DISCUSSION TITLE
labwc +waybar 下fcitx5能否正常工作？

### DISCUSSION THREAD

**Discussion by rico256-cn** [fcitx/fcitx5#695] [Q&A]
labwc 是一个 wayland 合成器，不知道在这下面waybar输入法状态显示， 以及labwc 下常规使用是否 ok？

**Comment by wengxt** [comment]
gtk 和 qt 和 wayland 都行吧，我在twitter 上看别人用过，text-input-v3 的不行，需要 labwc 支持对应的协议。
